packages/components/pie-webc-core/vite.config.js

```diff
@@ -18,7 +18,7 @@ export default defineConfig({
         globals: true,
         exclude: [
             '**/test/{accessibility,component,system,visual}/*.spec.{js,ts}',
-            '**/test/mixins/**/*.browser.spec.{js,ts}',
+            '**/test/*.browser.spec.{js,ts}',
```

Comment:
@siggerzz hey, I updated this path as now we have browser tests not only in mixins folder. I also removed this condition from root vite config. Do you have any objections?

Reply:
That's fine with me! 🎉